src/SFA.DAS.LearnerDataMismatches.Web/Pages/Learner.cshtml.cs

```diff
@@ -37,11 +38,13 @@ public class LearnerModel : PageModel
 
         private readonly IPaymentsDataContext context;
         private readonly ICommitmentsService commitmentsService;
+        private readonly IEmployerService employerService; 
 
-        public LearnerModel(IPaymentsDataContext context, ICommitmentsService commitmentsService)
+        public LearnerModel(IPaymentsDataContext context, ICommitmentsService commitmentsService, IEmployerService employerService)
```

Comment:
This is starting to get a bit busy, we may need to extract out some functionality if we add much more.